components/mail/dovecot/dovecot.p5m

```diff
@@ -0,0 +1,1055 @@
+file path=usr/include/dovecot/rfc822-parser.h
+file path=usr/include/dovecot/safe-memset.h
+file path=usr/include/dovecot/safe-mkdir.h
+file path=usr/include/dovecot/safe-mkstemp.h
+file path=usr/include/dovecot/sasl-server.h
+file path=usr/include/dovecot/sdbox-file.h
+file path=usr/include/dovecot/sdbox-storage.h
+file path=usr/include/dovecot/sdbox-sync.h
+file path=usr/include/dovecot/sendfile-util.h
+file path=usr/include/dovecot/seq-range-array.h
+file path=usr/include/dovecot/service-settings.h
+file path=usr/include/dovecot/settings-parser.h
+file path=usr/include/dovecot/settings.h
+file path=usr/include/dovecot/sha1.h
+file path=usr/include/dovecot/sha2.h
+file path=usr/include/dovecot/shared-storage.h
+file path=usr/include/dovecot/smtp-client.h
+file path=usr/include/dovecot/sort.h
+file path=usr/include/dovecot/sql-api-private.h
+file path=usr/include/dovecot/sql-api.h
+file path=usr/include/dovecot/sql-db-cache.h
+file path=usr/include/dovecot/ssl-proxy.h
+file path=usr/include/dovecot/stats-connection.h
+file path=usr/include/dovecot/stats-parser.h
+file path=usr/include/dovecot/stats.h
+file path=usr/include/dovecot/str-find.h
+file path=usr/include/dovecot/str-sanitize.h
+file path=usr/include/dovecot/str-table.h
+file path=usr/include/dovecot/str.h
+file path=usr/include/dovecot/strescape.h
+file path=usr/include/dovecot/strfuncs.h
+file path=usr/include/dovecot/strnum.h
+file path=usr/include/dovecot/subscription-file.h
+file path=usr/include/dovecot/syslog-util.h
+file path=usr/include/dovecot/test-common.h
+file path=usr/include/dovecot/time-util.h
+file path=usr/include/dovecot/timing.h
+file path=usr/include/dovecot/unichar.h
+file path=usr/include/dovecot/unix-socket-create.h
+file path=usr/include/dovecot/unlink-directory.h
+file path=usr/include/dovecot/unlink-old-files.h
+file path=usr/include/dovecot/uri-util.h
+file path=usr/include/dovecot/userdb-blocking.h
+file path=usr/include/dovecot/userdb-template.h
+file path=usr/include/dovecot/userdb-vpopmail.h
+file path=usr/include/dovecot/userdb.h
+file path=usr/include/dovecot/utc-mktime.h
+file path=usr/include/dovecot/utc-offset.h
+file path=usr/include/dovecot/var-expand.h
+file path=usr/include/dovecot/wildcard-match.h
+file path=usr/include/dovecot/write-full.h
+file path=usr/lib/dovecot/auth/libauthdb_imap.a
+file path=usr/lib/dovecot/auth/libauthdb_imap.so
+file path=usr/lib/dovecot/doveadm/lib10_doveadm_acl_plugin.a
+file path=usr/lib/dovecot/doveadm/lib10_doveadm_acl_plugin.so
+file path=usr/lib/dovecot/doveadm/lib10_doveadm_expire_plugin.a
+file path=usr/lib/dovecot/doveadm/lib10_doveadm_expire_plugin.so
+file path=usr/lib/dovecot/doveadm/lib10_doveadm_quota_plugin.a
+file path=usr/lib/dovecot/doveadm/lib10_doveadm_quota_plugin.so
+file path=usr/lib/dovecot/doveadm/lib20_doveadm_fts_plugin.a
+file path=usr/lib/dovecot/doveadm/lib20_doveadm_fts_plugin.so
+file path=usr/lib/dovecot/dovecot-config
+file path=usr/lib/dovecot/lib01_acl_plugin.a
```

Comment:
On Thu, Sep 08, 2016 at 20:46:37 -0700, Alexander Pyhalov wrote:
> > +file path=usr/include/dovecot/utc-offset.h
> > +file path=usr/include/dovecot/var-expand.h
> > +file path=usr/include/dovecot/wildcard-match.h
> > +file path=usr/include/dovecot/write-full.h
> > +file path=usr/lib/dovecot/auth/libauthdb_imap.a
> > +file path=usr/lib/dovecot/auth/libauthdb_imap.so
> > +file path=usr/lib/dovecot/doveadm/lib10_doveadm_acl_plugin.a
> > +file path=usr/lib/dovecot/doveadm/lib10_doveadm_acl_plugin.so
> > +file path=usr/lib/dovecot/doveadm/lib10_doveadm_expire_plugin.a
> > +file path=usr/lib/dovecot/doveadm/lib10_doveadm_expire_plugin.so
> > +file path=usr/lib/dovecot/doveadm/lib10_doveadm_quota_plugin.a
> > +file path=usr/lib/dovecot/doveadm/lib10_doveadm_quota_plugin.so
> > +file path=usr/lib/dovecot/doveadm/lib20_doveadm_fts_plugin.a
> > +file path=usr/lib/dovecot/doveadm/lib20_doveadm_fts_plugin.so
> > +file path=usr/lib/dovecot/dovecot-config
> > +file path=usr/lib/dovecot/lib01_acl_plugin.a
> 
> Do we need static libs?

No.  For whatever reason, I thought that there was a default transform that
dropped those.

> 
> -- 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/OpenIndiana/oi-userland/pull/2386/files/cd54caa068f1aa42c5d387e1866eb3f0b8d8e895#r78125229

-- 
#endif /* NO LIFE */